### PR TITLE
Redirect 301

### PR DESCRIPTION
**Pull request by wesanderson303**
@brianhyder Don't merge plz. Just a conversation starter until we finish our local 0.5.0 merge. This will redirect with status code 301. Defaults to 302 if no statusCode's provided.

### CODE REVIEW

File: include/http/request_handler.js [hunk 1]
```diff
@@ -91,6 +91,15 @@ module.exports = function RequestHandlerModule(pb) {
     RequestHandler.staticRoutes = {};
 
     /**
+     * Redirect http codes
+     * @static
+     * @property REDIRECT_CODES
+     * @type {Object}
+     */
+    RequestHandler.REDIRECT_CODES = {MOVED_PERMANENTLY: 301,
```
Comment (wesanderson303) [inline include/http/request_handler.js]: Is there a better place to place a static for status codes?

Comment (brianhyder) [inline include/http/request_handler.js]: Not something we want to keep up with.  I would use: https://www.npmjs.com/package/http-status-codes

Comment (brianhyder) [inline include/http/request_handler.js]: We could also add it to the requirements.js file so that all you have to do is: ```pb.HttpStatus.OK```

**Comment by wesanderson303** [comment]
@brianhyder I think this PR's all ready to go. Mind taking a look when you get some time?

**Pull request merged**